.travis.yml

```diff
@@ -1,16 +1,23 @@
 language: node_js
 node_js:
-- 8.9.0
+  - 8.9.0
 dist: trusty
 addons:
   chrome: stable
 branches:
   only:
-  - develop
-  - "/^feature\\/.*$/"
-before_script:
-- npm install -g angular-cli
-script: "./build.sh"
+    - develop
+    - "/^feature\\/.*$/"
+cache: yarn
```

Comment:
I believe @plabadie had some concerns about adding this. Can you check with him?